File: inputprint.java
// inputprint.java
import java.util.Scanner;

public class InputPrint {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter some text: ");
        String input = scanner.nextLine();
        System.out.println("You entered: " + input);
    }
}

File: inputprint.zig
// inputprint.zig
const std = @import("std");

pub fn main() !void {
    const stdout = std.io.getStdOut().writer();
    const stdin = std.io.getStdIn().reader();
    try stdout.print("Enter some text: ", .{});
    var input: [100]u8 = undefined;
    const inputStr = try stdin.readUntilDelimiterOrEof(input[0..], '\n');
    try stdout.print("You entered: {s}\n", .{inputStr});
}
